site/write_ups/2023/25-01-23_writeup.md

```diff
@@ -0,0 +1,108 @@
+
+Euan wants to find ways to introduce more Data Ethics material (and memes, of course) in lectures/tutorials for students and staff at his work. Euan also wants to successfully navigate academia collaboratively, rejecting the competition paradigm (slightly more ambitious than remembering to say "please" and "thank you" to your at home Google assistant!)
+
+Our very own Nina wants to find a way to spread the word about data ethics to other fields using data (e.g. psychology! medicine!) - this might be by running a data hazards workshop. She would also like to find a way to make this formally part of my job, rather than just a passion project. Nina would also like to be brave enough to engage more with the data ethics community, maybe by attending a conference?
+
+Vanessa wants to do a data ethics project with non-technical people, publish her data ethics work and make masters students contribute to her data ethics research.
+
+Jess wants to complete applied data science module and learn more about the technical side as well as get involved helping out with data ethics club!
+
+Ola wants to learn some tech/data visualisation skills to become a more well rounded person - Huw suggests the [JGI Data Viz seminar](https://dataviz.blogs.bristol.ac.uk/)! Ola will run a Digital Societies Reading group (late Feb-ish) on ground truth in ML: Kang, E. B. (2023), Ground truth tracings (GTT): On the epistemic limits of machine learning and Big Data & Society, 10(1). Precise details to be confirmed but everyone is invited so stay tuned!
+
+Noshin would like to think more about intersectionality in data and from whom's perspective is the data story being told. Holding the experience behind the data in mind when reporting on key themes surrounding children's social care. Also we don't do much peer review for our thematics, so it would be good to have this challenge! 
+
+Amy wants to use her publishing background to support our data ethics resolutions!
+
+## Club Resolutions
+
+Moving past individual goals, next we stopped to think about what we wanted the club itself! We generally liked the "The intro -> small group discussion - > full group sum-up" format of the meetings. Some suggestions included:
+
+- Try a weird voting system instead of first past the post 
+- Could we have an agony aunt for data ethics? Where people can bring specific data ethics conundrums they have?
+- Some more 'expert' presentations before our talks like we had for our discussion on [AI Generated Art](https://dataethicsclub.com/write_ups/2022/16-11-22_writeup.html) to give us more context to our discussions
+
+## How can you get involved?
+
+We work on Data Ethics Club as a passion project alongside their full time jobs (and sometimes, *instead of* their full time jobs). We are always happy to have people come and help us with all sorts of things to keep the club going, help us reach more people and provide better content for the group. Have a look below at how you can make an impact:
```

Comment:
```suggestion
The Data Ethics Club organisers work on this group, and all it's outputs, as a passion project alongside their full time jobs (and sometimes, *instead of* their full time jobs). We are always happy to have people come and help us with all sorts of things to keep the club going, help us reach more people and provide better content for the group. Have a look below at how you can make an impact:
```